tests/sentry/sentry_apps/external_requests/test_alert_rule_action_requester.py

```diff
@@ -1,10 +1,16 @@
 import responses
 
-from sentry.mediators.external_requests.alert_rule_action_requester import (
+from sentry.sentry_apps.external_requests.alert_rule_action_requester import (
     DEFAULT_ERROR_MESSAGE,
     DEFAULT_SUCCESS_MESSAGE,
     AlertRuleActionRequester,
 )
+
+# from sentry.mediators.external_requests.alert_rule_action_requester import (
+#     DEFAULT_ERROR_MESSAGE,
+#     DEFAULT_SUCCESS_MESSAGE,
+#     AlertRuleActionRequester,
+# )
```

Comment:
Could remove this too.

Reply:
oops, yeah that was debugging I forgot to remove